### ISSUE TITLE
Improve dmsgpty-ui logging when visor is no dmsgpty instance.

### ISSUE THREAD

**Issue by Senyoret1** [skycoin/skywire#212]
**Describe the bug**
When opening the dmsgpty-ui terminal for some visors, it works well, but for some others it just stays black.

**Environment information:**

- OS: Linux (Ubuntu 18.04.1)
- Platform: Linux 4.15.0-65-generic x86_64

**Steps to Reproduce**

1. Start some nodes using the `make integration-run-generic` command of the `skywire-services` repository (tested using the `milestone2` branch of the `skywire-mainnet` repo and the `mainnet-milestone2` branch of the  `skywire-services`  repo).

2. Open in a browser `https://{{ host }}/pty/024ec47420176680816e0406250e7156465e4531f5b26057c9f6297bb0303558c7` to open the the dmsgpty-ui terminal for VisorA. The terminal should work well on the browser and the command window for VisorA will show:
![va](https://user-images.githubusercontent.com/34079003/76146072-2e6c4680-6066-11ea-8b67-8d4d4d4e14f4.png)

3. Open in a browser `https://{{ host }}/pty/0348c941c5015a05c455ff238af2e57fb8f914c399aab604e9abb5b32b91a4c1fe` to open the the dmsgpty-ui terminal for VisorB. The terminal will stay black and the command window for VisorB will show:
![vb](https://user-images.githubusercontent.com/34079003/76146073-31ffcd80-6066-11ea-9bb9-6c269069178b.png)

**Actual behavior**
The terminal for VisorB does not work.

**Expected behavior**
The terminal should work.

**Additional context**

**Possible implementation**

**Comment by evanlinjin** [comment]
It means the visor has no dmsgpty instance associated with it. This is expected behaviour, although the logs can be clearer.

**Comment by Senyoret1** [comment]
Good :+1:. I reported this because I added a buton to the manager for opening the dmsgpty terminal on a new tab and had no way to know why the terminal showed nothing. In terms of functionality there is no big problem because the manager also has a simple terminal which works with the `exec` API endpoint, so it is possible to continue working if the visor does not have a dmsgpty instance associated with it.

In terms of the UX, it would be good to inform the user in the UI what happened, not just on the terminal were the visor is running. A way to do this could simply be to return an error text in the html obtained from the `/pty` URL, instead of the code for showing the terminal, which will not work, because the visor does not have a dmsgpty instance associated with it.
